airflow/kubernetes/pod_generator.py

```diff
@@ -62,6 +63,13 @@ def add_init_container(self,
     def _get_init_containers(self):
         return self.init_containers
 
+    def add_port(self, port):
+        """
+        Args:
```

Comment:
Instead of the Google style, we're using the sphinx docstring style in the Airflow project :-)

https://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html